standards/technical-approach.md

```diff
@@ -16,24 +36,56 @@ TADs do not have to be created for all development work, but MUST be created in
 - If the scope of work is expected to extend beyond a single sprint
 - If multiple developers will be contributing to the new feature
 - If the pros and cons of introducing a new tool or technology to a project, team, or the department need to be considered
+- If developers disagree or are unsure on the entire approach for implementing the new feature. Generally, a TAD should eliminate the kind of pull request that generates feedback in the realm of, "We need to rethink this entire approach."
 
 TADs SHOULD be created in the following situations:
 
 - If multiple approaches to a feature or problem are valid and the engineer assigned the ticket(s) wishes to document the decisions made
 - If questions are raised about the appropriateness of an approach and engineer/team wishes to document the decision process
-- If a engineer is faced with a decision or choice that they would like to receive feedback on and/or have validated by a more senior member of the engineering team
+- If a engineer is faced with a decision or choice that they would like to receive feedback on and/or have validated by a more senior member of the engineering team.
+- If writing the TAD will save engineering team time in the implementation phase. Figuring out the hardest challenges in a TAD usually helps saving time down the road.
+
+## Resourcing
+
+The quarterly planning process is a good time for Tech Leads and Architects to look ahead at the upcoming quarter and ensure enough engineering time is allocated to writing TADs for a planned feature or project.
+
+## Timing
+
+The TAD MUST be written before any implementation work begins. However, TADs SHOULD NOT be written _too far_ in advance of implementation. Best practice is to make Jira tickets and start implementation in the sprint immediately following document approval.
+
+## Proof of Concepts
+
+A TAD MAY include a proof of concept as demonstration of feasibility of and confidence in the recommended approach.
+
+A proof of concept should be time-boxed to no longer than one sprint (two weeks). Tech leads and architects SHOULD guide engineers in generating a list of questions that the proof of concept aims to answer or clarify.
+
+A proof of concept MUST NOT go to production. The expectation is that a proof of concept is a rough draft built using mock data and may not include the cleanest or most maintainable code. If possible, it is recommended to implement a solution that _only_ runs locally & in a separate repository with a name that includes `proof of concept` in the title OR in a feature branch that is left unmerged.
+
+## TAD Template
```

Comment:
@oliviawongnyc Added this section about the template, which I do think is helpful. Lmk what you think.

Reply:
Excellent! TY!